src/view/adminhtml/layout/default.xml

```diff
@@ -16,5 +16,10 @@
                 <argument name="template" xsi:type="string">ScandiPWA_Customization::page/footer.phtml</argument>
             </action>
         </referenceBlock>
+        <referenceBlock name="version">
+            <action method="setTemplate">
+                <argument name="template" xsi:type="string">ScandiPWA_Customization::page/header.phtml</argument>
+            </action>
+        </referenceBlock>
```

Comment:
Please try the third option from [this](https://dolphinwebsolution.com/how-to-include-add-custom-js-in-magento-2/) page instead and move your JS to the `.js` file.
This would be a more appropriate solution than adding an additional `.phtml` file.